src/content/app/genome-browser/components/track-panel/components/track-panel-list/TrackPanelList.test.tsx

```diff
@@ -44,30 +46,56 @@ jest.mock('./track-panel-items/TrackPanelRegularItem', () => () => (
 ));
 
 const mockState = createMockBrowserState();
+const activeGenomeId = mockState.browser.browserGeneral.activeGenomeId;
 
-const mockStore = configureMockStore([thunk]);
+const server = setupServer(
+  rest.get('http://track-api/track_categories/:genomeId', (req, res, ctx) => {
+    const genomeId = req.params.genomeId as string;
+    if (genomeId === activeGenomeId) {
+      const mockData = { track_categories: createGenomeCategories() };
 
-let store: ReturnType<typeof mockStore>;
+      return res(ctx.json(mockData));
+    }
+  })
+);
 
 const renderComponent = (state: typeof mockState = mockState) => {
-  store = mockStore(state);
-  return render(
+  const store = configureStore({
+    reducer: createRootReducer(),
+    middleware: (getDefaultMiddleware) =>
+      getDefaultMiddleware().concat([restApiSlice.middleware]),
```

Comment:
This line doesn't seem to be formatted properly. Shouldn't it be starting from two spaces to the right?

Reply:
It's the return value from the function; it's like a child of line 65.